files/install_puppet.ps1

```diff
@@ -30,14 +30,17 @@ param(
   # correctly from the module.
   [parameter(Position=0)]
   [String] $PuppetPID,
-  [String] $Source='<%= @_msi_location %>',
-  [String] $Logfile='<%= @_logfile %>',
-  [String] $InstallDir='<%= @install_dir %>',
-  [String] $PuppetMaster='<%= @_puppet_master %>',
-  [String] $PuppetStartType='<%= @_agent_startup_mode %>',
-  [String] $InstallArgs='<%= @_install_options   %>',
-  [String] $InstallScriptPIDFile='<%= @_install_pid_file_loc %>',

```

Comment:
This isn't in the PS based set of args? Is that on purpose? From reading below, it looks like you removed the need for it, but wanted to double check.

Reply:
yeah, there's no need for this param anymore